kzz7yh-f57075/cod-ve09v2_kzz7yh-f57075.xml

```diff
@@ -55,13 +55,13 @@
          <div xml:id="kzz7yh-f57075"><!-- l2d19a3 -->
             <head xml:id="kzz7yh-f57075-Hd1e101">Articulus 3</head>
             <p xml:id="kzz7yh-f57075-d1e103">
-               <lb ed="#V" n="120"/>Consequenter queritur de tertio 
+               <lb ed="#V" n="120"/>COnsequenter queritur de tertio 
       prin<lb ed="#V" n="121" break="no"/>cipali. Et circa hoc 
       <lb ed="#V" n="122"/>queruntur duo.
    </p>
             <p xml:id="kzz7yh-f57075-d1e112">
       ¶ Primo vtrum illa 
-      immortali<lb ed="#V" n="123" break="no"/>talitas qua adam posset non mori inesset sibi per 
+      immortali<lb ed="#V" n="123" break="no"/>talitas <!--Now what the heck is this word--> qua adam posset non mori inesset sibi per 
```

Comment:
You've got it correct as it exists in the text, but it is definitely a printing error so i wrapped it in `<sic>`
```suggestion
      <sic>immortali<lb ed="#V" n="123" break="no"/>talitas</sic> qua adam posset non mori inesset sibi per 
```